src/Stratis.Bitcoin.Features.PoA/Voting/Poll.cs

```diff
@@ -52,5 +57,17 @@ public void ReadWrite(BitcoinStream stream)
                 this.PubKeysHexVotedInFavor = arr.ToList();
             }
         }
+
+        public override string ToString()
+        {
+            string keysVotedInFavor = "";
+            foreach (string keyHex in this.PubKeysHexVotedInFavor)
```

Comment:
you could replace this `foreach` with `string.Join(" ", this.PubKeysHexVotedInFavor);` or `this.PubKeysHexVotedInFavor.Aggregate((curr, next) => $"{curr} {next}");` That will also eliminate trailing whitespace

Reply:
fixed